src/g_editor.c

```diff
@@ -2397,7 +2397,9 @@ static void canvas_doclick(t_canvas *x, int xpos, int ypos, int which,
                 {
                     if (doit)
                     {
-                        int issignal = obj_issignaloutlet(hitobj, closest);
+                        int nchannels = obj_sigoutletgetchans(hitobj, closest);
+                        int connection_thickness = (nchannels > 1 ? MC_CONNECTION_WIDTH : nchannels == 1 ? 2.0 : 1.0) * x->gl_zoom;
```

Comment:
nitpick: use the same indentation as in the other two places to avoid long lines.

Also, you can actually write `2` and `1` instead of `2.0` and `1.0`. We don't need doubles here. My mistake.